.ci/wercker.sh

```diff
@@ -152,6 +152,20 @@ no-error-strata)
   rm -rf Strata
   ;;
 
+no-error-spring-integration)
+  set -e
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  checkout_from https://github.com/spring-projects/spring-integration.git
+  cd .ci-temp/spring-integration
+  PROP_MAVENLOCAL="mavenLocal"
+  PROP_CSVERSION="checkstyleVersion"
```

Comment:
extra variables to avoid spellchecker violations and extending our whitelist.
```
$ ./.ci/test-spelling-unknown-words.sh 
Clean up from previous run
Run w
Review results
New misspellings found, please review:
Pcheckstyle
Pmaven

If you are ok with the output of this run, you will need to
patch .ci/jsoref-spellchecker/whitelist.words <<EOF
@@ -999,2 +999,3 @@
 pbludov
+Pcheckstyle
 pdf
@@ -1017,2 +1018,3 @@
 plugin
+Pmaven
 pmd
EOF
```

Reply:
Please add to whitelist.words `MAVENLOCAL` and `CSVERSION` too.

Reply:
`_` were added to named to bypass adding to whiltelist